From 517eda52bebb6a0c5df06b52ff81a19ed9b71740 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Piotr=20W=C3=B3jcik?= <chocimier@tlen.pl>
Date: Wed, 8 Nov 2017 12:41:22 +0100
Subject: [PATCH] Use newer minimal version of taglib

CMake rejects 1.11.1 as "greater" than 1.5, so let's use 1.10 as minimal
so it accepts.
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git CMakeLists.txt CMakeLists.txt
index f0fd6d5..c18e9f1 100644
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -17,9 +17,9 @@ INCLUDE (MacroLibrary)
 INCLUDE (FindGettext)
 
 # Find taglib
-set(TAGLIB_MIN_VERSION "1.5")
+set(TAGLIB_MIN_VERSION "1.10")
 macro_optional_find_package(Taglib REQUIRED)
-macro_log_feature( TAGLIB_FOUND "taglib" "Read and write tags in audio files" "http://www.freshmeat.net/projects/taglib" TRUE "1.5.0" "")
+macro_log_feature( TAGLIB_FOUND "taglib" "Read and write tags in audio files" "http://www.freshmeat.net/projects/taglib" TRUE "1.10.0" "")
 
 IF(TAGLIB_FOUND)
   ADD_DEFINITIONS( -DWITH_TAGLIB )
-- 
2.15.0

